src/components/Home/TextArea/Mention/AutoCompleteMentionList.tsx

```diff
@@ -0,0 +1,42 @@
+import { Separator } from "@/components/ui/separator.tsx";
+
+import { cn } from "@/lib/utils.ts";
+import { MyType } from "@/constants/dummyData.ts";
+
+export interface ListProps {
+  users: MyType[];
+  onClick: (people: MyType) => void;
+}
+
+interface MentionListProps extends ListProps {
+  focusIndex: number;
+}
+
+const AutoCompleteMentionList = ({ users, onClick, focusIndex }: MentionListProps) => {
+  if (!users.length) return "";
+  return (
+    <div className="mt-2 overflow-hidden scroll-auto border p-2">
+      {users.map(({ name, userId }, idx) => {
+        return (
+          <ul
+            key={userId}
+            className={cn("hover:bg-gray-100", focusIndex === idx ? "bg-gray-100" : "")}
+          >
+            <li
+              onClick={() => onClick({ name, userId })}
+              className={cn(
+                "cursor-pointer py-2 hover:font-bold",
+                focusIndex === idx ? "font-bold" : "",
+              )}
+            >
+              {name}
+            </li>
+            {idx !== users.length - 1 && <Separator />}
+          </ul>
```

Comment:
헉 이렇게 되면 li 하나당 ul 하나를 갖게 되는 것 같아요! 아래처럼 변경해주셔야 할 것 같습니다 : ) 바꾸시는김에 `idx`도 `index`로 바꿔주시면...감사하겠습니다..ㅎ🥺
```suggestion
    <ul className="mt-2 overflow-hidden scroll-auto border p-2">
      {users.map(({ name, userId }, idx) => {
        return (
          <>
            <li key={userId} className={cn()} onClick={() => onClick({ name, userId })}>
              {name}
            </li>
            {idx !== users.length - 1 && <Separator />}
          </>
        );
      })}
    </ul>
```

Reply:
허걱ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 제가 마음이 급했나봐요ㅠㅠ 바로 수정해서 다시 올릴게요!! 감사합니다🥰🥰